apps/src/templates/lessonOverview/activities/ProgressionDetails.jsx

```diff
@@ -33,7 +32,7 @@ export default class ProgressionDetails extends Component {
         : scriptLevel.levels[0];
 
     return {
-      status: LevelStatus.not_tried,
+      id: -1,
```

Comment:
@cforkish checking in on this one. This seems like it should be `activeLevel.id` rather than -1. Do you remember why you wrote it this way?